### PR TITLE
Suite style fix2

### PR DESCRIPTION
**Pull request by Ayagoumi**
This PR includes the following changes:

- Updated the style of the date selector.
Screenshot 2023-02-09 at 14 10 23
Screenshot 2023-02-09 at 14 04 02

- Removed the USD conversion.
Screenshot 2023-02-09 at 14 10 51
Screenshot 2023-02-09 at 14 10 47
Screenshot 2023-02-09 at 14 10 40

- Removed AddDelegator related files because they were unused.
- Updated the font weight of the Validator part in the section.
Screenshot 2023-02-09 at 14 14 40

### CODE REVIEW

File: src/components/misc/AvaxInput.vue [hunk 1]
```diff
@@ -20,10 +20,6 @@
                     <b>{{ $t('misc.balance') }}:</b>
                     {{ balanceBig }}
                 </p>
-                <p>
-                    <b>$</b>
-                    {{ amountUSD.toLocaleString(2) }}
```
Comment (Chriserus) [inline src/components/misc/AvaxInput.vue:25]: Can we delete `amountUSD` from this file entirely?

File: src/components/misc/CurrencyInputDropdown.vue [hunk 2]
```diff
@@ -15,7 +15,6 @@
                         :placeholder="placeholder"
                         :disabled="disabled"
                     ></big-num-input>
-                    <p class="usd_val" :active="isAvax">${{ amountUSD.toLocaleString(2) }}</p>
```
Comment (Chriserus) [inline src/components/misc/CurrencyInputDropdown.vue:18]: Same here

File: src/components/wallet/portfolio/FungibleRow.vue [hunk 3]
```diff
@@ -18,11 +18,6 @@
             <span>
                 {{ amtBig.toLocaleString() }}
             </span>
-            <br />
-            <span class="fiat" v-if="isAvaxToken">
-                {{ totalUSD.toLocaleString(2) }}
```
Comment (Chriserus) [inline src/components/wallet/portfolio/FungibleRow.vue:23]: Same here

File: src/components/wallet/transfer/FormC.vue [hunk 4]
```diff
@@ -60,9 +60,6 @@
                     {{ $t('transfer.fee_tx') }}
                     <span>{{ maxFeeText }} {{ nativeAssetSymbol }}</span>
                 </p>
-                <p>
-                    <span>${{ maxFeeUSD.toLocaleString(2) }} USD</span>
```
Comment (Chriserus) [inline src/components/wallet/transfer/FormC.vue:64]: And here

File: src/views/wallet/Transfer.vue [hunk 5]
```diff
@@ -57,10 +57,6 @@
                             {{ $t('transfer.fee_tx') }}
                             <span>{{ txFee.toLocaleString(9) }} {{ nativeAssetSymbol }}</span>
                         </p>
-                        <p>
-                            {{ $t('transfer.total_native') }}
-                            <span>{{ totalUSD.toLocaleString(2) }} USD</span>
```
Comment (Chriserus) [inline src/views/wallet/Transfer.vue:62]: And here

**Review by aeddaqqa — APPROVED** [review]
LGTM

**Review by Chriserus — APPROVED** [review]
LGTM

**Pull request merged**